Juriba.Dashworks/Public/Get-DwImportDeviceFeed.ps1

```diff
@@ -39,21 +39,18 @@ function Get-DwImportDeviceFeed {
         [string]$Instance,
         [Parameter(Mandatory=$true)]
         [string]$APIKey,
-        [parameter(Mandatory=$false, ParameterSetName="ImportId")]
```

Comment:
one of the reasons for using parameter sets is that they allow us to prevent both ImportId and Name being passed to the function. does this function still work ok if both parameters are passed?